docs/metrics.md

```diff
@@ -0,0 +1,466 @@
+
+::: note
+Enabling `Queues.Processing` can cause a lot more time series to be generated, according to the `QueueLabels` and `MessageTypes` configuration.
+:::
+
+Example Configuration:
+```
+Queues": {
+	"Busy": true,
+	"Length": true,
+	"Processing": false
+}
+
+"QueueLabels": [
+	{
+		"Regex": "StorageReaderQueue #.*",
+		"Label": "Readers"
+	},
+	{
+		"Regex": ".*",
+		"Label": "Others"
+	}
+]
+```
+
+Example Output:
+```
+# TYPE eventstore_queue_busy_seconds counter
+# UNIT eventstore_queue_busy_seconds seconds
+eventstore_queue_busy_seconds{queue="Readers"} 1.04568158125 1688157491029
+eventstore_queue_busy_seconds{queue="Others"} 0 1688157491029
+
+# TYPE eventstore_queue_queueing_duration_max_seconds gauge
+# UNIT eventstore_queue_queueing_duration_max_seconds seconds
+eventstore_queue_queueing_duration_max_seconds{name="Readers",range="16-20 seconds"} 0.06434454 1688157489545
+eventstore_queue_queueing_duration_max_seconds{name="Others",range="16-20 seconds"} 0 1688157489545
+```
+
+### Status
+
+EventStoreDB can track the current status of the `Node`, the `Index`, and the `Scavenge`
+
+| Time Series | Type       | Description |
+| :---------- | :--------- | :---------- |
+| `eventstore_statuses{name=<NAME>,status=<STATUS>}` | [Gauge](#common-types) | Number of seconds since the 1970 epoch when _NAME_ most recently had the status _STATUS_ |
+
+For a given _NAME_, the current status can be determined by taking the max of all the time series with that name.
+
+`Index` can have one of the following statuses:
+- Opening (loading/verifying the PTables)
+- Rebuilding (indexing previously written records on start up)
+- Initializing (initializing any other parts of the index e.g. StreamExistenceFilter on start up)
+- Merging
+- Scavenging
+- Idle
+
+`Scavenge` can have one of the following statuses:
+- Accumulation
+- Calculation
+- Chunk Execution
+- Chunk Merging
+- Index Execution
+- Cleaning
+- Idle
+
+`Node` can have one of the node statuses: `Leader`, `Follower`, etc.
+
+Example Configuration:
+```
+"Statuses": {
+	"Index": true,
+	"Node": false,
+	"Scavenge": false
+}
+```
+
+Example Output:
+```
+# TYPE eventstore_checkpoints gauge
+eventstore_statuses{name="Index",status="Idle"} 1688054162 1688054162477
+```
+
+### Storage Writer
+
+| Time Series | Type       | Description |
+| :---------- | :--------- | :---------- |
+| `eventstore_writer_flush_size_max{range=<RANGE>}`             | [RecentMax](#recentmax) | Recent maximum flush size in bytes       |
+| `eventstore_writer_flush_duration_max_seconds{range=<RANGE>}` | [RecentMax](#recentmax) | Recent maximum flush duration in seconds |
+
+Example Configuration:
+```
+"Writer": {
+	"FlushSize": true,
+	"FlushDuration": false
+}
+```
+
+Example Output:
+```
+# TYPE eventstore_writer_flush_size_max gauge
+eventstore_writer_flush_size_max{range="16-20 seconds"} 410 1688056823193
+```
+
+### System
```

Comment:
put this one near the `Process` as they are closely related

Reply:
currently its alphabetical, i'd probably keep it that way but open to persuasion

Reply:
is ok alphabetical